go.mod

```diff
@@ -188,3 +190,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.3 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/rancher/shepherd => github.com/susesgartner/shepherd v0.0.0-20250326154529-beddceca67fa
```

Comment:
Talked offline about this, but putting blocking comment here to make sure we don't merge with this forked reference.